ml_dtypes/tests/finfo_test.py

```diff
@@ -16,6 +16,7 @@
 from absl.testing import parameterized
 import ml_dtypes
 import numpy as np
+from multi_thread_test_mixin import MultiThreadTestMixin
```

Comment:
I worry about defining local import paths for test utilities, because it causes issues in some testing environments. Could we define this in `ml_dtypes/_test_utils.py` so we can import it from the installed package?

Reply:
@jakevdp Alternatively, I can add `ml_dtypes/tests/__init__.py`  and replace absolute import with a relative:
```python
from .multi_thread_test_mixin import MultiThreadTestMixin
```

What do you think, will it solve the import issue for those testing envs ?

Reply:
That's fine, though I'd suggest absolute rather than relative imports
```python
from ml_dtypes.tests.multi_thread_test_mixing import MultiThreadTestMixin
```